advanced-healthcare-materials.csl

```diff
@@ -0,0 +1,150 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-US">
+  <info>
+    <title>Advanced Healthcare Materials</title>
+    <title-short>AHM</title-short>
+    <id>http://www.zotero.org/styles/advanced-healthcare-materials</id>
+    <link href="http://www.zotero.org/styles/advanced-healthcare-materials" rel="self"/>
+    <link href="http://www.zotero.org/styles/angewandte-chemie" rel="template"/>
+    <link href="http://onlinelibrary.wiley.com/journal/10.1002/(ISSN)2192-2659/homepage/2087_authorresources.html" rel="documentation"/>
+    <author>
+      <name>Patrick O'Brien</name>
+    </author>
+    <category citation-format="numeric"/>
+    <category field="engineering"/>
+    <category field="chemistry"/>
+    <issn>2192-2640</issn>
+    <eissn>2192-2659</eissn>
+    <summary>A style for Wiley-VCH's journal "Advanced Healthcare Materials". This style has many limits due to csl constraints, most notably the inability to include pages/page ranges for books, the wrong square brackets for two items cited together, and the lack of support for citing 1a)... b)...c)....</summary>
```

Comment:
> This style has many limits due to csl constraints, most notably the inability to include pages/page ranges for books, the wrong square brackets for two items cited together, and the lack of support for citing 1a)... b)...c)....

@POBrien333, can you elaborate a little on the first two points?

Reply:
1) is E. Wingender, Gene Regulation in Eukaryotes, Wiley-VCH, Weinheim, Germany 1993, **p. 215**. 
i.e. the idea of citing individual pages of a book in a numeric style. Not sure how to capture that. 2) I'm also curious about.